yarn-project/boxes/blank-react/package.json

```diff
@@ -15,6 +15,7 @@
     "formatting": "prettier --check ./src && eslint ./src",
     "formatting:fix": "prettier -w ./src",
     "compile": "aztec-cli compile src/contracts --outdir ../artifacts --typescript ../artifacts",
+    "compile:local": "ts-node --esm ../cli/src/bin/index.ts compile \"src/contracts\" --outdir \"src/artifacts\" --typescript \"src/artifacts\"",
```

Comment:
Think the paths should be relative `--outdir ../artifacts --typescript ../artifacts`. Sorry I didn't run it while typing it down to you.